typescript/utils/src/types.ts

```diff
@@ -5,6 +5,7 @@ export enum ProtocolType {
   Ethereum = 'ethereum',
   Sealevel = 'sealevel',
   Cosmos = 'cosmos',
+  Starknet = 'starknet',
```

Comment:
since we're adding a new protocol type - automatically i would class this PR as a major change and not minor or patch